charger/openevse.go

```diff
@@ -83,47 +81,26 @@ func NewOpenEVSE(uri, user, password string, timeout time.Duration) (api.Charger
 
 	var phases1p3p func(int) error
 
-	ok, err := c.IsChargingOnThreePhases()
+	ok, err := c.isChargingOnThreePhases()
```

Comment:
I don't understand. Does OpenEVSE always have phase switching support? Then we don't need the decorator. We _only_ need to ensure that we control switching.

Reply:
It's a kit, so some EVSEs will, some won't, depending on firmware you upload.
I have two contactors in mine to support phase switching.

Reply:
Again: the check needs to be for the capability, not for the fact _if_ its currently charging 3p!

Reply:
I think I understand.